MobileCenterAnalytics/MobileCenterAnalytics/Support/MobileCenterAnalytics.xcconfig

```diff
@@ -1,3 +1,3 @@
 #include "../../../Global.xcconfig"
 
-OTHER_LDFLAGS = $(inherited) -framework Foundation -framework UIKit -framework CoreTelephony;
```

Comment:
@TroubleMakerBen 
We need sqlite dependency only for test projects because each test project links with libMobileCenter.a.
I moved dependency to linker flags for test projects.